x/wasm/internal/keeper/keeper.go

```diff
@@ -566,6 +566,10 @@ func (k Keeper) dispatchMessages(ctx sdk.Context, contractAddr sdk.AccAddress, m
 
 func gasForContract(ctx sdk.Context) uint64 {
 	meter := ctx.GasMeter()
+	// avoid integer overflow
```

Comment:
It avoids an underflow, when `meter.GasConsumed() > meter.Limit()`, comment should be corrected.